app/controllers/users/sessions_controller.rb

```diff
@@ -91,13 +93,21 @@ def cache_active_profile
     end
 
     def user_signed_in_and_not_locked_out?(user)
-      return false unless current_user.present?
-
+      return false unless current_user
       !user_locked_out?(user)
```

Comment:
This is not new to this PR. I didn't want to make any changes that were not related. I can add the `.present?` back if that makes things clearer.